pkg/cloud/exoscale/actuators/cluster/actuator.go

```diff
@@ -190,7 +189,7 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) error {
 
 // GetIP returns IP address of the machine in the cluster.
 func (*Actuator) GetIP(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
-	klog.Infof("Getting IP of machine %v for cluster %v.", machine.Name, cluster.Name)
+	klog.Infof("Getting IP of the machine %v for cluster %v.", machine.Name, cluster.Name)
 
 	machineStatus, err := exoscalev1.MachineSpecFromMachineStatus(machine.Status.ProviderStatus)
```

Comment:
imho, this name should read `machineStatusFromProviderStatus`

Reply:
👍